scripts/openshiftci-e2e-4x-psi-tests_mac.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+export SENDQUEUE=amqp.ci.queue.mac.e2e.send
+export SENDTOPIC=amqp.ci.topic.mac.e2e.send
+
+. scripts/openshift-e2e-4x-psi-tests.sh
```

Comment:
As per our meeting today, can we merge this and openshift-e2e-4x-psi-tests_win.sh together and pass win or mac as parameter values to set SENDQUEUE and SENDTOPIC based on that?
This will reduce the number of scripts